connector/mqtt/vernemq/README.md

```diff
@@ -0,0 +1,316 @@
+-H "Authorization: Bearer ${JWT}" \
+-H "Content-Type:application/json" \
+-H "Accept:application/json" \
+-d  "{\"certificate\": \"${CSR_CONTENT}\", \"passwd\": \"dojot\"}" | jq '.status.data' -r)
+```
+
+##### Create a public certificate
+
+Create *client.crt* file from EJBCA return
+
+```console
+(echo  "-----BEGIN CERTIFICATE-----"
+echo ${CRT_ENTITY_CONTENT}
+echo "-----END CERTIFICATE-----" ) > client.crt
+```
+
+#### 8. Obtaining the root certificate (public certificate of **CA**)
+
+##### Retrieve CA Certificate
+
+Retrieve certificate and set in a environment variable
+
+```console
+export CRT_ROOT_CONTENT=$(curl  -X GET ${DOJOT_URL}/ca/${CA_NAME} \
+-H "Authorization: Bearer ${JWT}" \
+-H "Content-Type:application/json" \
+-H "Accept:application/json" | jq '.certificate' -r )
+```
+
+##### Create a file certificate from Root
+
+Create a root certificate *root.crt* file from EJBCA return
+
+```console
+(echo  "-----BEGIN CERTIFICATE-----"
+echo ${CRT_ROOT_CONTENT}
+echo "-----END CERTIFICATE-----" ) > root.crt
+```
+
+## Simulating a device with mosquitto
+
+NOTE: Ports and address may change depending on the deployment
+
+NOTE 2: The **tenant** is *admin* and **device_id** is *a1998e* for these examples. You must change them for your case.
+
+In these examples we will use [mosquitto](https://mosquitto.org/) client.
+
+##### Some options for mosquitto_pub and mosquitto_sub
+
+- **-h**: Specify the host to connect to. Defaults to localhost. In this case: *myhost*.
+
+- **-p**: Connect to the port specified. If not given, the default of 1883 for plain MQTT or 8883 for MQTT over TLS will be used. In this case: *30311*.
+
+- **-t**: The MQTT topic on which to publish the message. See mqtt(7) for more information on MQTT topics. In this case: *admin:a1998e/attrs* to publish and *admin:a1998e/config* to subscription.
+
+- **--cafile**: Define the path to a file containing PEM encoded CA certificates that are trusted. Used to enable SSL communication. In this case: *root.crt*
+
+- **--cert**: Define the path to a file containing a PEM encoded certificate for this client, if required by the server. In this case: *client.crt*
+
+- **--key**: Define the path to a file containing a PEM encoded private key for this client, if required by the server. In this case: *client.key*
+
+- **-m**: Send a single message from the command line.
+
+- **-u**: Provide a username to be used for authenticating with the broker.
+
+### With security: use TLS to communicate with VerneMQ
+
+The tree files are require: *client.crt*, *client.key* and *root.crt*. See example:
+
+##### Example on how to publish
+
+```console
+mosquitto_pub -h myhost -p 8883 -t admin:a1998e/attrs -m '{"attr_example": 10 }' --cert client.crt  --key client.key --cafile root.crt
+```
+
+Note: In this case, the message is a publish  on an attribute with the label attr_example and a new value 10 in device *a1998e* with tenant *admin*.
```

Comment:
In this case, the message is a publish on an attribute
Doubt: In this case, the message is a publish with an attribute?